### PR TITLE
Update AdministratorCommands.pwn for guide ( could someone merge the guide requests pls )

### PR DESCRIPTION
**Pull request by Specifer**
Requesting Implementation for #518 , With Modification for /show Discord and guide

### CODE REVIEW

File: pawn/Elements/AdministratorCommands.pwn [hunk 1]
```diff
@@ -972,6 +972,16 @@ lvp_show(playerId, params[]) {
         showInfo = true;
         format(g_message, sizeof(g_message), "Save money and stats! Register your nickname at www.sa-mp.nl");
     }
+    
+    else if (!strcmp(showParameter, "guide", true, 5)) {
```
Comment (RussellLVP) [inline pawn/Elements/AdministratorCommands.pwn]: I don't think this is particularly appropriate for `/show`, as very few people get temporary rights. There's no need to broadcast this to everyone on the server, is there?

File: pawn/Elements/AdministratorCommands.pwn [hunk 2]
```diff
@@ -899,7 +899,7 @@ GodHelp:
         ColorManager->storeExistingPlayerCustomColor(subjectId);
         ColorManager->setPlayerCustomColor(subjectId, Color::AdministratorColor);
 
-        SendClientMessage(subjectId, Color::Success, "You have been granted administrator rights.");
+        SendClientMessage(subjectId, Color::Success, "You are now a temporary administrator. Kindly read the contents of /guide");
```
Comment (RussellLVP) [inline pawn/Elements/AdministratorCommands.pwn:902]: This seems good once the other PRs have been submitted. Thank you.

**Review by RussellLVP — CHANGES_REQUESTED** [review]

File: pawn/Elements/AdministratorCommands.pwn (hunk 1 above)
Comment (Specifer) [reply pawn/Elements/AdministratorCommands.pwn]: Yes sure .

File: pawn/Elements/AdministratorCommands.pwn [hunk 3]
```diff
@@ -973,14 +973,10 @@ lvp_show(playerId, params[]) {
...
-    else if (!strcmp(showParameter, "guide", true, 5)) {
-        showInfo = true;
-        format(g_message, sizeof(g_message), "Newly Temp'ed on LVP . Are you in use of Guidance , Check Out /guide for Preliminary Notes and Measures ");
-    }
-    
+ 
     else if (!strcmp(showParameter, "discord", true, 7)) {
         showInfo = true;
-        format(g_message, sizeof(g_message), "Join us on Official LVP Discord Server at www.discord.sa-mp.nl ");
+        format(g_message, sizeof(g_message), "Join us on official LVP Discord Server at discord.sa-mp.nl");
```
Comment (Specifer) [inline pawn/Elements/AdministratorCommands.pwn]: Modified the past line here .

File: pawn/Elements/AdministratorCommands.pwn [hunk 4]
```diff
@@ -973,14 +973,10 @@ lvp_show(playerId, params[]) {
         format(g_message, sizeof(g_message), "Save money and stats! Register your nickname at www.sa-mp.nl");
     }
     
-    else if (!strcmp(showParameter, "guide", true, 5)) {
-        showInfo = true;
-        format(g_message, sizeof(g_message), "Newly Temp'ed on LVP . Are you in use of Guidance , Check Out /guide for Preliminary Notes and Measures ");
-    }
-
```
Comment (Specifer) [inline pawn/Elements/AdministratorCommands.pwn]: Removed /show guide

**Comment by RussellLVP** [comment]
This looks good to me once we have introduced `/guide`.

**Pull request closed without merging**